### PR TITLE
Issue 58

### PR DESCRIPTION
**Pull request by VAO11**
@womenhackfornonprofits/whfnp
## Linked Jira/Github Issue Ticket

[58](https://github.com/womenhackfornonprofits/whfnp/issues/58)
## If the changes are visual, provide a screenshot.

non-profit-contact-form_iss58

@lili2311 Hi Lili, can you please review my change for issue 58 and let me know if you are happy for this to be merged

### CODE REVIEW

**Comment by lili2311** [comment]
I think we need to move the form the very top so that it is not something you need to scroll for, could you add it to the section right under the "What to expect" so that the form is on white background and is right underneath where we explain how it works? And for now leave Start Your Journey Here section as we still want to keep that page.

**Comment by tanyapowell** [comment]
Hey @VAO11 😄  - just wondering why so many existing files are coming up as new files in your PR? E.g. the `gruntfile.js` and `README.md`

File: src/non-profits.html [hunk 1]
```diff
@@ -78,8 +78,39 @@ <h2 class="section-heading">Do you have any questions?</h2>
...
+      <label for="email">Email</label>
+      <input type="email" name="_replyto" placeholder="Laura@example.com" required aria-required="true" class="form-control">
+    </div>
+    <div class="form-group">
+      <label for="contact-tel">Phone <span class="help-text">(optional)</span></label>
+      <input type="tel" name="contact-tel" placeholder="07912 345 678" class="form-control">
+    </div>
+    <div class="form-group">
+      <label for="project-description">What kind of technical help would you like?</label>
+      <textarea name="project-description" rows="3" placeholder="Website, mobile, app" class="form-control"></textarea>
```
Comment (tanyapowell) [inline src/non-profits.html:101]: Think `app` can be misleading. To a client an `app` could mean a mobile application, but to a techie that could mean a desktop app, web app or mobile app. Think it's best to explicitly say what you mean to reduce potential confusion

File: src/non-profits.html [hunk 2]
```diff
@@ -78,8 +78,39 @@ <h2 class="section-heading">Do you have any questions?</h2>
...
+    </div>
+    <div class="form-group">
+      <label for="project-value">How will this project help you? <span class="help-text">(optional)</span></label>
+      <textarea name="project-value" rows="3" placeholder="Volunteer outreach, revamp the website" class="form-control"></textarea>
```
Comment (tanyapowell) [inline src/non-profits.html:105]: `Volunteer outreach` is also a confusing example to have. Maybe something like `improving our online presence`

File: src/non-profits.html [hunk 3]
```diff
@@ -78,8 +78,39 @@ <h2 class="section-heading">Do you have any questions?</h2>
 <section>
   <h2 class="section-heading">Start your journey here</h2>
   <p>Please fill in the form below to see how we can help bring your project to life.
```
Comment (tanyapowell) [inline src/non-profits.html:80]: Something doesn't seem right with this line. Think the copy needs improving

@krissy or @clawey you are wordsmiths - what do you think?

**Comment by tanyapowell** [comment]
Also, @VAO11 your placeholders in the image above don't match the ones in the file. Could you paste in an updated image. Ta

**Comment by VAO11** [comment]
@tanyapowell
I only changed the non-profits.html file. Not sure how I managed to the other files pulled in to be honest.

@lili2311 
I can move the form higher up th e page np.

I can either cancel this pull request and delete the branch to start again or I will need some guidance to clean up the branch only include the changes I made.

I this the other file changes were done by Lili to get the workspace working from c9.io.

What would you suggest?

File: src/non-profits.html (hunk 3 above)
Comment (lili2311) [inline src/non-profits.html:80]: I think get rid of it completely to be honest, or get a completely new line. You are right it doesn't sound right at all.

**Comment by lili2311** [comment]
Sure thing we can tidy up, it looks like perhaps a jekyll serve command run in a wrong directory and created a`_site` file in the root of the project. So on your branch go to the root folder for `whfnp` and delete the `_site` folder completely. Do a `git status` and `git add`, then `git commit` and `git push`. This should remove this unwanted folder

**Comment by VAO11** [comment]
Closing Branch as too many additional changes. Creating a new Branch.

**Pull request closed without merging**